run_node.sh

```diff
@@ -79,55 +106,42 @@ get_snapshot () {
     if [ ! -d "${DB_SNAPSHOT_PATH}/db/full" ] && [ -z "$SYNC" ]
     then
         echo "Downloading the snapshot..."
-        pushd "${DB_SNAPSHOT_PATH}"
-        wget -O ${DB_SNAPSHOT_FILE} ${DB_SNAPSHOT_URL}
+        pushd "${DB_SNAPSHOT_PATH}" > /dev/null
+        wget -q -O ${DB_SNAPSHOT_FILE} ${DB_SNAPSHOT_URL}
         tar xvzf ${DB_SNAPSHOT_FILE}
         rm ${DB_SNAPSHOT_FILE}
-        popd
+        popd > /dev/null
     fi
 }
 
 get_chainspec () {
     # For testnet and mainnet we get the chainspec from the repo (with the commit identified by the hash we get from System::version RPC call)
     echo "Downloading the chainspec..."
-    pushd "${HOST_BASE_PATH}"
-    wget -O ${CHAINSPEC_FILE} https://raw.githubusercontent.com/Cardinal-Cryptography/aleph-node/"${ALEPH_VERSION}"/bin/node/src/resources/${CHAINSPEC_FILE}
-    popd
+    pushd "${HOST_BASE_PATH}" > /dev/null
+    wget -q -O ${CHAINSPEC_FILE} https://raw.githubusercontent.com/Cardinal-Cryptography/aleph-node/"${ALEPH_VERSION}"/bin/node/src/resources/${CHAINSPEC_FILE}
+    popd > /dev/null
 }
 
 get_docker_image () {
     echo "Pulling docker image..."
     ALEPH_IMAGE=public.ecr.aws/p6e8q1z1/aleph-node:${ALEPH_VERSION:0:7}
-    docker pull "${ALEPH_IMAGE}"
+    docker pull --quiet "${ALEPH_IMAGE}"
```

Comment:
Maybe some info log after that `docker image pulled success`, as now this is `quiet`.